schema/staticwebapp.config.json

```diff
@@ -0,0 +1,613 @@
+      "patternProperties": {
+        ".*": {
+          "oneOf": [
+            {
+              "type": "object",
+              "properties": {
+                "redirect": {
+                  "type": "string",
+                  "description": "Redirect to a relative or absolute path, or an external URI. Default status code is 302, override with 301."
+                },
+                "statusCode": {
+                  "type": "integer",
+                  "description": "Status code"
+                },
+                "rewrite": {
+                  "type": "string",
+                  "description": "A path to rewrite the request route to"
+                }
+              }
+            }
+          ]
+        }
+      }
+    },
+    "mimeTypes": {
+      "type": "object",
+      "description": "Custom mime types configuration",
+      "default": {},
+      "examples": [
+        {
+          ".config": "application/xml"
+        }
+      ],
+      "patternProperties": {
+        "^\\..+$": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "globalHeaders": {
+      "type": "object",
+      "description": "Default headers to set on all responses",
+      "additionalProperties": true
+    },
+    "auth": {
+      "$ref": "#/definitions/auth"
+    },
+    "networking": {
+      "type": "object",
+      "description": "Networking configuration",
+      "properties": {
+        "allowedIpRanges": {
+          "type": "array",
+          "description": "Restrict access to one or more IPv4 ranges. Supports CIDR notation (e.g., \"192.168.100.14/24\")",
+          "items": {
+            "type": "string"
+          },
+          "examples": [["10.0.0.0/24", "192.1.1.1/10"]]
+        }
+      },
+      "additionalProperties": false
+    },
+    "forwardingGateway": {
+      "type": "object",
+      "description": "Forwarding gateway configuration",
+      "properties": {
+        "allowedForwardedHosts": {
+          "type": "array",
+          "description": "The value of `X-Forwarded-Host` to allow to be used when generating redirect URLs",
+          "items": {
+            "type": "string"
+          },
+          "examples": [["example.org", "www.example.org", "staging.example.org"]]
+        },
+        "requiredHeaders": {
+          "type": "object",
+          "description": "HTTP header name/value pairs that are required for access",
+          "examples": [
+            {
+              "X-Azure-FDID": "10dd26ef"
+            }
+          ],
+          "additionalProperties": true
+        }
+      },
+      "additionalProperties": false
+    },
+    "platform": {
+      "type": "object",
+      "description": "Platform configuration",
+      "properties": {
+        "apiRuntime": {
+          "type": "string",
+          "enum": [
+            "dotnet:3.1",
+            "dotnet:6.0",
+            "dotnet-isolated:6.0",
+            "dotnet-isolated:7.0",
+            "node:12",
```

Comment:
Latest platforms seems to be missing compared to Azure Functions: 
`az functionapp list-runtimes --os linux --query "[].{stack:join(':', [runtime, version]), LinuxFxVersion:linux_fx_version, SupportedFunctionsVersions:to_string(supported_functions_versions[])}" --output table`

Reply:
Will investigate the details, thanks!

Reply:
Have updated supported platform "dotnet-isolated:8.0" in PR https://github.com/Azure/static-web-apps-cli/pull/809.